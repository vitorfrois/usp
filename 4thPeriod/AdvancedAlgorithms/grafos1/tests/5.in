10 20 10
5 6 8
3 7 1
3 8 9
8 10 2
4 9 1
2 4 2
1 3 10
2 3 10
7 10 7
4 5 6
1 2 5
9 10 1
6 7 10
3 9 10
6 9 6
3 4 9
6 10 6
7 8 5
1 5 2
8 9 8
7 1
6 5
6 7
5 3
3 2
4 5
6 4
4 9
4 7
10 9
