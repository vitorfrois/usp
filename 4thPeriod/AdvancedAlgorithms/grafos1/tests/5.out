11
8
10
12
10
6
7
1
9
1
